Examples/Reminders/Schema.swift

```diff
@@ -41,10 +57,10 @@ extension Reminder {
 
 extension Reminder.TableColumns {
   var isPastDue: some QueryExpression<Bool> {
-    !isCompleted && #sql("coalesce(date(\(dueDate)) < date('now'), 0)")
+    !isCompleted && #sql("coalesce(date(\(dueDate)) < now(), 0)")
```

Comment:
When [this](https://github.com/pointfreeco/swift-structured-queries/pull/61) lands we could potentially stay in our builder:

```suggestion
    !isCompleted && dueDate.map { $0 < Date() }
```

I know we like SQL to do as much work as possible, but maybe this is a case where it's more convenient and easier to control for tests by staying in Swift.